salt/states/pkg.py

```diff
@@ -2329,12 +2329,16 @@ def latest(
     '''
     refresh = salt.utils.pkg.check_refresh(__opts__, refresh)
 
+    if not name:
+        # Note state checks name is a string, this should only happen if their is a bug
+        raise CommandExecutionError('Parameter name must contain label or package name.')
+
```

Comment:
I don't understand why you added this. It doesn't seem like it is necessary, for one thing, and it would result in an ugly traceback in the state results if it were ever triggered.